### ISSUE TITLE
Add array trait to ipysheet columns

### ISSUE THREAD

**Issue by DougRzz** [QuantStack/ipysheet#57]
Ipysheet looks great!  I have been trying to find a way to use it in some of my projects.  I would love to jslink a couple of ipysheet columns with a bqplot scatter plot.  I was hoping this would work.... 
```python
widgets.jslink((column2, 'value'), (scatt, 'x'))
widgets.jslink((column3, 'value'), (scatt, 'y'))
```
where scatt is a scatter mark .  However, the scatter x, y attributes  require numpy arrays.  Would it be possible to add an 'array' trait to the ipysheet columns so they could be syncable with bqplot marks?

**Comment by martinRenou** [comment]
Thanks for the report! Do you have an error message when you do this? I think this should work out of the box. I'll try

**Comment by martinRenou** [comment]
Ok, I could reproduce your issue.

One workaround that you could use would be to do a `directional_link` from the column to the plot:
```python
from traitlets import directional_link

directional_link((column2, 'value'), (scatt, 'x'))
```

And for the other way you can use an explicit callback that converts the numpy array to a list:
```python
def handle_plot_change(*arg, **kwargs):
    column2.value = scatt.x.tolist()
    
scatt.observe(handle_plot_change, names=['x'])
```

It's more typing, but that should do the job until a next version with a fix is out :)

**Comment by DougRzz** [comment]
Thanks. Ipysheet is really nice.  To be honest, initially I was struggling to see the point of it.  It just seemed like a grid for positioning widgets - defaulting to a text box.   Now i see the potential! :)

```python
import numpy as np
import pandas as pd
from bqplot import (
    Axis, ColorAxis, LinearScale, DateScale, DateColorScale, OrdinalScale,
    OrdinalColorScale, ColorScale, Scatter, Lines, Figure, Tooltip
)
from ipywidgets import Label

## Get Data
dataLen = 14

price_data = pd.DataFrame(np.cumsum(np.random.randn(dataLen, 2).dot([[1.0, -0.8], [-0.8, 1.0]]), axis=0) + 100,
                          columns=['Security 1', 'Security 2'], index=pd.date_range(start='01-01-2007', periods=dataLen))
size = dataLen
np.random.seed(0)
x_data = range(size)
y_data = np.cumsum(np.random.randn(size) * 100.0)
ord_keys = np.array(['A', 'B', 'C', 'D', 'E', 'F'])
ordinal_data = np.random.randint(5, size=size)

symbols = ['Security 1', 'Security 2']

dates_all = price_data.index.values
dates_all_t = dates_all[1:]
sec1_levels = np.array(price_data[symbols[0]].values.flatten())
log_sec1 = np.log(sec1_levels)
sec1_returns = log_sec1[1:] - log_sec1[:-1]

sec2_levels = np.array(price_data[symbols[1]].values.flatten())

# sc_x = DateScale()
sc_x = LinearScale()
sc_y = LinearScale()

scatt = Scatter(x=x_data, y=sec2_levels, scales={'x': sc_x, 'y': sc_y},enable_move = True)
ax_x = Axis(scale=sc_x, label='Date')
ax_y = Axis(scale=sc_y, orientation='vertical', tick_format='0.0f', label='Security 2')

fig = Figure(marks=[scatt], axes=[ax_x, ax_y],  animation_duration = 2000)

import ipysheet
import ipywidgets as widgets
sheet = ipysheet.sheet(rows=len(scatt.x), columns=2)

# column1 = ipysheet.column(0, [widgets.FloatSlider()]*len(scatt.x))

column2 = ipysheet.column(0, list(scatt.x))
column3 = ipysheet.column(1, list(scatt.y))

# for row in range(len(scatt.x)):
#     slider = widgets.FloatSlider()
#     cell = ipysheet.cell(row, 0,slider, style={'min-width': '150px'})
#     cellInCol2 = ipysheet.cell(row, 1)
#     widgets.jslink((cellInCol2, 'value'), (slider, 'value'))

def handle_plot_changex(*arg, **kwargs):
    column2.value = scatt.x.tolist()
    
scatt.observe(handle_plot_changex, names=['x'])

def handle_plot_changey(*arg, **kwargs):
    column3.value = scatt.y.tolist()
    
scatt.observe(handle_plot_changey, names=['y'])

from traitlets import directional_link

directional_link((column2, 'value'), (scatt, 'x'))
directional_link((column3, 'value'), (scatt, 'y'))

# for row in range(len(scatt.x)):
#     slider = widgets.FloatSlider()
#     cell = ipysheet.cell(row, 0,slider, style={'min-width': '150px'})
#     cellInCol2 = ipysheet.cell(row, 1)
#     widgets.jslink((cellInCol2, 'value'), (slider, 'value'))



widgets.HBox([fig,sheet])

```
![ipysheet_bqplot](https://user-images.githubusercontent.com/3510318/53746325-d91cb880-3e98-11e9-93f7-733e33dc1b6c.gif)

**Comment by martinRenou** [comment]
That is nice! Thanks for sharing your example :D

**Comment by maartenbreddels** [comment]
Amazing example as always by @DougRzz !

**Comment by maartenbreddels** [comment]
#63 supports arrays, but it converts them in the validator, not sure I like the approach yet.

**Comment by martinRenou** [comment]
This has been fixed in `0.3.1`
